browser.js

```diff
@@ -14,11 +14,14 @@ if (crypto && crypto.getRandomValues) {
 
 function randomBytes (size, cb) {
   // in browserify, this is becomes an extended UInt8Array
-  var bytes = new Buffer(size)
+  var rawBytes = new global.UInt8Array(size)
 
   // This will not work in older browsers.
   // See https://developer.mozilla.org/en-US/docs/Web/API/window.crypto.getRandomValues
-  crypto.getRandomValues(bytes)
+  crypto.getRandomValues(rawBytes)
```

Comment:
so it needs to be an integer based typed array, anything that starts with Int or Uint and this dues fix it